play-with-git/src/w2/sistem_inscrieri/GuestList.java

```diff
@@ -0,0 +1,103 @@
+package w2.sistem_inscrieri;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+public class GuestList implements GuestListActions {
+    private final int locuriDisponibile;
+    private ArrayList<Guest> participanti;
+    private LinkedList<Guest> inAsteptare;
```

Comment:
Maybe to use a combination of a hashMap and a List to improve the get performance or something similar? 
What do you think @alexandruzarnoianu  about it?